pkg/webui/console/components/pubsub-form/validation-schema.js

```diff
@@ -91,10 +91,10 @@ export default Yup.object().shape({
         otherwise: Yup.string().strip(),
       }),
       subscribe_qos: Yup.string()
-        .oneOf(qosLevels, sharedMessages.validateFormat)
+        .oneOf(qosLevels)
```

Comment:
<img width="1080" alt="Screenshot 2020-04-21 at 15 20 32" src="https://user-images.githubusercontent.com/16374166/79865532-f8034480-83e3-11ea-9c13-3ee61c100b4e.png">

the `required` requirement is never checked